### ISSUE TITLE
Regression: Error_Handler: CRITICAL: {'detail': 'The required query parameter granularity was not specified.', 'error': 400, 'errorCode': 'MISSING_QUERY_PARAMETER', 'parameters': ['granularity'], 'reason': 'Bad Request'}

### ISSUE THREAD

**Issue by mgmonteleone** [mgmonteleone/python-atlasapi#178]
test_12_issue_90_get_measurement_for_host
and
test_13_get_multiple_metrics_at_once_for_host

now fails with

```python
Error_Handler: CRITICAL: {'detail': 'The required query parameter granularity was not specified.', 'error': 400, 'errorCode': 'MISSING_QUERY_PARAMETER', 'parameters': ['granularity'], 'reason': 'Bad Request'}
```

Could be related to recent changes in #176 ?

**Comment by mgmonteleone** [comment]
Can not reproduce now.

**Comment by mgmonteleone** [comment]
Reproduced, was related to #176... need to add back param handling.

Params were removed from the URI (old way of doing), this then broke other methods which did not yet send the params in the new method. Testing and fixing.

**Comment by mgmonteleone** [comment]
Fixed and released in v3.0.1b5
